Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -628,25 +628,30 @@ def genS(fragment,howMuch):
 U2000by1={'--relval': '2000,1'}
 U80by1={'--relval': '80,1'}
 
-hiAlca = {'--conditions':'auto:run2_mc_hi', '--era':'Run2_2016,Run2_HI'}
 hiAlca2011 = {'--conditions':'auto:run1_mc_hi'}
+hiAlca2015 = {'--conditions':'auto:run2_mc_hi', '--era':'Run2_2016,Run2_HI'}
+hiAlca2018 = {'--conditions':'auto:phase1_2017_realistic', '--era':'Run2_2017'}
```

Comment:
The proper "2018" is not currently necessarily best supported.
 @kpedro88 may know better about the state of phase-1 HCAL.